src/main/java/seedu/address/logic/parser/AddEventCommandParser.java

```diff
@@ -45,7 +45,7 @@ public AddEventCommand parse(String args) throws ParseException {
                 argMultiMap.getValue(PREFIX_EVENT_START_DATE).get(),
                 argMultiMap.getValue(PREFIX_EVENT_END_DATE).get());
 
-        Event event = new Event(eventName, eventDescription, eventDuration);
+        Event event = new Event(eventName, eventDescription, eventDuration, -1);
```

Comment:
You could consider overloading the constructor method of Event here, such that the ID is initialized to -1 if not specified. This way, your code could be clearer, and the original constructor method signature can still be used.

Reply:
Thanks for the feedback! I will change it right now.